src/helpers/colorHelpers.ts

```diff
@@ -0,0 +1,31 @@
+export const rainbowStop = (val:string) => {
+  const h:number = parseFloat(val)
+  const f = (n:number, k = (n + h * 12) % 12) =>
+    0.5 - 0.5 * Math.max(Math.min(k - 3, 9 - k, 1), -1);
+  const rgb2hex = (r:number, g:number, b:number) =>
```

Comment:
Pls extract `rgb2hex` from `rainbowStop`, it might be helpful to have it at other cases.